app/interactors/create_group.rb

```diff
@@ -1,2 +1,13 @@
 class CreateGroup < StandardInteraction
+  def validate_input
+    context.fail!(errors: "invalid input") unless context.group_params
```

Comment:
So I think the new error handling in #95 covers this a bit better. But I suppose I could override `initialize` with more explicit arguments i.e.
```ruby
initialize(user_params, id)
  ...
end
```
to maybe throw an argument error if it's not called with the correct arguments.